packages/shared/src/components/cards/squad/SquadGrid.tsx

```diff
@@ -53,79 +43,62 @@ const borderColorToClassName: Record<SourceCardBorderColor, string> = {
 
 export const SquadGrid = ({
   source,
-  title,
-  subtitle,
-  icon,
-  action,
-  description,
-  borderColor = SourceCardBorderColor.Avocado,
-  banner,
-}: SourceCardProps): ReactElement => {
+}: UnFeaturedSquadCardProps): ReactElement => {
```

Comment:
Simplified the component to just base the content from a Squad since we don't need the extra props anymore.